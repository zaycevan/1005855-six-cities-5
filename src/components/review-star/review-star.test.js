import renderer from "react-test-renderer";
import ReviewStar from "./review-star";

const noop = () => {};

const tree = renderer.create(
    <ReviewStar
      onChange={noop}
      id={1}
      title={`perfect`}
      disabled={false}
      rating={`4`}
    />
);

it(`Should ReviewStar render correctly`, () => {
  expect(tree.toJSON()).toMatchSnapshot();
});
