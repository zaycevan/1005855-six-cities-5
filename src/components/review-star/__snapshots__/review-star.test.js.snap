// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should ReviewStar render correctly 1`] = `
Array [
  <input
    className="form__rating-input visually-hidden "
    disabled={false}
    id="1-stars"
    name="rating"
    onChange={[Function]}
    type="radio"
    value={1}
  />,
  <label
    className="reviews__rating-label form__rating-label"
    htmlFor="1-stars"
    title="perfect"
  >
    <svg
      className="form__star-image"
      height="33"
      width="37"
    >
      <use
        xlinkHref="#icon-star"
      />
    </svg>
  </label>,
]
`;
